src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    public Circle(String color) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to assign the value to a variable and then pass it to the constructor for better code readability.